### PR TITLE
feat: add keyboard navigation to svelte inspector and improve a11y

### PR DESCRIPTION
**Pull request by dominikg**
this PR follows up on #436 to complete keyboard only navigation and a11y improvements

cc @geoffrich

### CODE REVIEW

File: packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte [hunk 1]
```diff
@@ -222,9 +257,11 @@
 			.join('')}`}
 		on:click={() => toggle()}
 		bind:this={toggle_el}
+		aria-label={`${enabled ? 'disable' : 'enable'} svelte-inspector`}
```
Comment (geoffrich) [inline packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte:260]: `aria-label` won't have any effect on a div. Can this be a `button` instead?

**Review by geoffrich — COMMENTED** [review]
One small comment. The announcer stuff _seems_ like it should work, but you'd probably want to test with VoiceOver or something to be sure.

File: packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte (hunk 1 above)
Comment (dominikg) [reply packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte:260]: yes, button is very possible. thanks for noticing

Comment (dominikg) [reply packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte:269]: is a button now, thank you `all: unset`

**Comment by dominikg** [comment]
i also cleaned up the changeset files, hoping it creates a cleaner changelog for 1.1.0. given inspector is still experimental and is going to move into a separate package at a later stage, should we use patch instead?

**Comment by bluwy** [comment]
I think it'd be fine if we stick as a patch 👍

**Review by bluwy — APPROVED** [review]

**Pull request merged**